Formula/rocksdb5.rb

```diff
@@ -0,0 +1,80 @@
+class Rocksdb5 < Formula
```

Comment:
Is it newly written or copied from anywhere (if so — shan't it retain the license)?

Reply:
Created PR #2